include/units/core.h

```diff
@@ -2508,8 +2511,8 @@ namespace units
 			std::enable_if_t<detail::is_time_conversion_factor<Cf> && detail::is_losslessly_convertible<Rep, T>, int> =
 				0>
 		constexpr unit(const std::chrono::duration<Rep, Period>& value) noexcept
-		  : linearized_value(NumericalScale::linearize(units::convert<unit>(
-				units::unit<units::conversion_factor<Period, dimension::time>, Rep>(value.count()))()))
+		  : linearized_value(NumericalScale::linearize(units::convert<unit>(units::unit<units::conversion_factor<Period, dimension::time>, Rep>(value.count()))

```

Comment:
This line is longer than the Clang Format limit. Why don't you turn it on to run on every save?

Reply:
it's on, I think visual studio's implemention isn't very good. I've seen it crap out a few times.

Reply:
I've been looking into automating it in a Travis build. I'll get back to you when I'm successful.